var/spack/repos/builtin/packages/neko/package.py

```diff
@@ -27,6 +28,7 @@ class Neko(AutotoolsPackage, CudaPackage, ROCmPackage):
     version("develop", branch="develop")
     variant("parmetis", default=False, description="Build with support for parmetis")
     variant("xsmm", default=False, description="Build with support for libxsmm")
+    variant("gslib", default=False, description="Build with support for gslib")
```

Comment:
Have you considered making this a conditional variant (see https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants)?

Reply:
Indeed, should definitely be a conditional variant.